src/MetaModels/Helper/ToolboxFile.php

```diff
@@ -924,6 +926,16 @@ private function processFile($fileName)
             $information['src'] = $fileName;
         }
 
+        // Prepare the picture for provide the image size.
+        if ($information['isPicture'] = (bool) ((int) $this->resizeImages[2])) {
```

Comment:
Why double casting?

Reply:
I would like a Boolean for isPicture. Do you have another idea how to get a boolean here?
I've got a string here instead of an integer.

Reply:
Ok, that explains it. Would be a better comment though.
I can see from the code that `// Prepare the picture for provide the image size.` is being done.
However it would be better to have `// Cast the string value image size to boolean.` therefore I asked.
All fine then (you might want to alter the comment though).

Reply:
I have change